### ISSUE TITLE
Application fails with ModuleNotFoundError: No module named 'onelogin.api.client'

### ISSUE THREAD

**Issue by rslima** [physera/onelogin-aws-cli#157]
## Expected Behavior
The application should at least show a version message

## Current Behavior
the application now fails with the same stack trace no matter which options I put on the command line
```
Traceback (most recent call last):
  File "/home/ricardo/.local/bin/onelogin-aws-login", line 5, in <module>
    from onelogin_aws_cli.cli import login
  File "/home/ricardo/.local/lib/python3.10/site-packages/onelogin_aws_cli/__init__.py", line 15, in <module>
    from onelogin.api.client import OneLoginClient
ModuleNotFoundError: No module named 'onelogin.api.client'
```

## Possible Solution

The OneLogin library is now on version 3.0.0 maybe the API have changed and the code should be adapted to it

## Steps to Reproduce

1. Just run the application on the command line in a new environment

**Comment by EricChea** [comment]
@rslima looks like there is a breaking change because of a major release of the `onelogin` package.  Looks like this package is not maintained much.  The fix is to lock onelogin to 2.0.4.  I just uninstalled (`pip uninstall onelogin`) and reinstalled (`pip install onelogin==2.0.4`).

**Comment by rslima** [comment]
It worked. Thanks

**Comment by markatgrindr** [comment]
I've also been facing this issue.  Quick patch would be to explicitly specify version in the requirements.txt: 2.0.4 or else `< 3.0.0

**Comment by slycoder** [comment]
@markatgrindr If you're up for it, you can submit a PR and we can get that change merged and pushed.

**Comment by slycoder** [comment]
Published a new version (hopefully it worked!)

**Comment by yasuno0327** [comment]
Hello, @slycoder.
It looks like not fixed this issue yet.
I installed v0.18.0.
But, installed onelogin package is still 3.0.0.

```bash
> pip install onelogin-aws-cli
Collecting onelogin-aws-cli
  Downloading onelogin_aws_cli-0.1.18-py3-none-any.whl (11 kB)
Requirement already satisfied: boto3 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from onelogin-aws-cli) (1.23.0)
Collecting onelogin
  Downloading onelogin-3.0.0-py3-none-any.whl (358 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 358.1/358.1 kB 1.9 MB/s eta 0:00:00
Requirement already satisfied: keyring in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from onelogin-aws-cli) (23.5.0)
Requirement already satisfied: requests in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from onelogin-aws-cli) (2.27.1)
Requirement already satisfied: botocore<1.27.0,>=1.26.0 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from boto3->onelogin-aws-cli) (1.26.10)
Requirement already satisfied: s3transfer<0.6.0,>=0.5.0 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from boto3->onelogin-aws-cli) (0.5.2)
Requirement already satisfied: jmespath<2.0.0,>=0.7.1 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from boto3->onelogin-aws-cli) (1.0.0)
Requirement already satisfied: importlib-metadata>=3.6 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from keyring->onelogin-aws-cli) (4.11.3)
Requirement already satisfied: urllib3>=1.25.3 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from onelogin->onelogin-aws-cli) (1.26.9)
Requirement already satisfied: python-dateutil in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from onelogin->onelogin-aws-cli) (2.8.2)
Requirement already satisfied: certifi>=2017.4.17 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from requests->onelogin-aws-cli) (2021.10.8)
Requirement already satisfied: charset-normalizer~=2.0.0 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from requests->onelogin-aws-cli) (2.0.12)
Requirement already satisfied: idna<4,>=2.5 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from requests->onelogin-aws-cli) (3.3)
Requirement already satisfied: zipp>=0.5 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from importlib-metadata>=3.6->keyring->onelogin-aws-cli) (3.8.0)
Requirement already satisfied: six>=1.5 in /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages (from python-dateutil->onelogin->onelogin-aws-cli) (1.16.0)
Installing collected packages: onelogin, onelogin-aws-cli
Successfully installed onelogin-3.0.0 onelogin-aws-cli-0.1.18

[notice] A new release of pip available: 22.2.2 -> 22.3.1
[notice] To update, run: pip install --upgrade pip

> pip show onelogin
ame: onelogin
Version: 3.0.0
Summary: OneLogin API
Home-page: https://onelogin.com/
Author: OneLoginDevex
Author-email: support@onelogin.com
License: MIT
Location: /Users/yoshitomo.yasuno/.pyenv/versions/3.10.4/lib/python3.10/site-packages
Requires: python-dateutil, urllib3
Required-by: onelogin-aws-cli
```

I think need to update install_requires in setup.py.

```diff
# in setup.py
install_requires=[
        'boto3',
-        'onelogin' 
+        'onelogin>=2.0,<3.0',
        'keyring',
        'requests',
],
```

**Comment by slycoder** [comment]
@yasuno0327 Pushed out your fix as 1.19
